src/main/java/com/moddy/server/service/model/ModelService.java

```diff
@@ -202,4 +224,31 @@ public void updateOfferAgreeStatus(Long offerId){
 
     }
 
+    @Transactional
+    public UserCreateResponse createModel(String baseUrl, String code, ModelCreateRequest request) {
+
+        String kakaoId = kakaoSocialService.getIdFromKakao(baseUrl, code);
+
+        Model model = Model.builder()
+                .name(request.name())
+                .year(request.year())
+                .gender(request.gender())
+                .phoneNumber(request.phoneNumber())
+                .isMarketingAgree(request.isMarketingAgree())
+                .profileImgUrl(DEFAULT_PROFILE_IMG_URL)
+                .kakaoId(kakaoId)
+                .role(Role.MODEL)
+                .build();
+
+        modelJpaRepository.save(model);
+
+        request.preferRegions().stream().forEach(preferRegionId -> {
+            Region region = regionJpaRepository.findById(preferRegionId).orElseThrow(() -> new NotFoundException(ErrorCode.NOT_FOUND_REGION_EXCEPTION));
+            PreferRegion preferRegion = PreferRegion.builder().user(model).region(region).build();
+            preferRegionJpaRepository.save(preferRegion);
+        });
```

Comment:
```suggestion
        request.preferRegions().stream().forEach(this::a);
    }
    private a(Long preferRegionId) {
            Region region = regionJpaRepository.findById(preferRegionId).orElseThrow(() -> new NotFoundException(ErrorCode.NOT_FOUND_REGION_EXCEPTION));
            PreferRegion preferRegion = PreferRegion.builder().user(model).region(region).build();
            preferRegionJpaRepository.save(preferRegion);
    }
```
p3
이런 식으로 내부 로직을 함수로 빼면 stream 코드를 더욱 단축할 수 있습니다.